packages/components/phone-number/src/components/osds-phone-number/osds-phone-number.scss

```diff
@@ -1,3 +1,35 @@
 @import '~@ovhcloud/ods-common-theming/color/ods-theming-color';
 @import '~@ovhcloud/ods-common-theming/ods-theme';
 
+:host(.phone-number) {
```

Comment:
is the host declaration necessary?
can't you put this directly under `.phone-number`?

Reply:
No It's not working just un the `.phone-number` 😢